src/diffusers/models/controlnetxs.py

```diff
@@ -817,9 +817,7 @@ def increase_block_input_in_encoder_resnet(unet: UNet2DConditionModel, block_no,
     norm_kwargs = {a: getattr(old_norm1, a) for a in norm_args}
     norm_kwargs["num_channels"] += by  # surgery done here
     # conv1
-    conv1_args = (
-        "in_channels out_channels kernel_size stride padding dilation groups bias padding_mode lora_layer".split(" ")
-    )
+    conv1_args = "in_channels out_channels kernel_size stride padding dilation groups bias padding_mode".split(" ")
```

Comment:
Can we condition this by making use of `USE_PEFT_BACKEND`? 

https://github.com/huggingface/diffusers/blob/664e931bcbddd47c21c3d4d547221506f588b9e9/src/diffusers/loaders/lora.py#L414